### PR TITLE
Policy fixes for federated reporting i2/i3

### PR DESCRIPTION
**Pull request by olehermanse**
See: https://github.com/cfengine/system-testing/pull/213

### CODE REVIEW

**Comment by olehermanse** [comment]
@craigcomstock @nickanderson Feel free to review this. Will wait for API changes before I can do more here and eventually get everything tested and merged. (So it's not done, hence the WIP label).

File: cfe_internal/enterprise/federation/federation.cf [hunk 1]
```diff
@@ -361,6 +366,8 @@ bundle agent entry
 bundle agent setup_status
 {
 @if minimum_version(3.11)
```
Comment (nickanderson) [inline cfe_internal/enterprise/federation/federation.cf]: @olehermanse I see that `bundle agent setup_status` is called for any case where federated reporting is `enabled`.  If we want to support 3.10 feeders this needs to be re-worked to not use the `with` attribute (introduced in 3.11).

Do you want me to open a separate PR or push to your fork or ...?

Comment (nickanderson) [reply cfe_internal/enterprise/federation/federation.cf]: Well, here is a PR against your PR.
https://github.com/olehermanse/masterfiles/pull/1

Comment (olehermanse) [reply cfe_internal/enterprise/federation/federation.cf]: Thanks!
